web/templates/pages/member_page.html

```diff
@@ -0,0 +1,39 @@
+{% extends 'base.html' %}
+
+{% load staticfiles %}
+
+{% block title %}TrackCat Member{% endblock %}
+
+{% block content%}
+
+	<div class="row">
+		<div class="member-profile">
+			<div class="col-md-12">
+			<h2>{% if user.fullname %}
+					{{ user.fullname }} 
+				{% else %}
```

Comment:
This is still not indented properly.
There is no harm in starting the `{% if %}` sentence in new line.